issue_07/script.js

```diff
@@ -0,0 +1,49 @@
+function startLoading() {
+    const div = document.getElementById('js-loading');
+    const loadingImg = document.createElement('img');
+    loadingImg.src = 'loading-circle.gif';
+    loadingImg.alt = 'ローディング画像';
+    div.appendChild(loadingImg);
+}
+
+function stopLoading() {
+    document.getElementById('js-loading').remove();
+}
+
+function createLists(listsData) {
+    const ul = document.getElementById('js-ul');
+    const fragment = document.createDocumentFragment();
+    for (const item of listsData) {
+        const li = document.createElement('li');
+        const p = document.createElement('p');
+        const img = document.createElement('img');
+
+        p.textContent = item.text;
+        img.src = item.img;
+        img.alt = item.alt;
+
+        fragment.appendChild(li).appendChild(p).appendChild(img);
+    }
+    ul.appendChild(fragment);
+}
+
+startLoading();
+
+const getData = new Promise((resolve) => {
+    const lists = [{
```

Comment:
```suggestion
const getData = new Promise((resolve) => {
　startLoading();
    const lists = [{
```
It would be good to run it before the promise in this function since it will be loading for 3 seconds before the promise is resolved.

Reply:
I had been mulling over the starting position of the function, but now it is clear.
Thank you!
https://github.com/yoshikawa-akr/js-lessons/pull/7/commits/cdcb4c4a228780edf7bc84ade390da09df23ef0b